digitalaidseattle/seattle-humane-app
# Feature/1.2.6.2/top menu modification

## What type of PR is this? (check all applicable)


## Description
- Removed side nav bar for more real-estate for the dashboard and other pages
- Change styling and coloring based on Figma design and Set theme colors
- Make sure accessibility and responsiveness are intact and added rules to accommodate smaller screens

## Figma
[Wire Frame](https://www.figma.com/file/BboJuNpVaUcHlA8ELX5MOu/Seattle-Humane-DAS?type=design&node-id=10%3A430&mode=design&t=jqZZZMfHgite3o7w-1)

# QA Instructions, Screenshots, Recordings

## Screenshots
### Smaller screen view
![Screenshot 2024-03-13 at 1 32 15 AM](https://github.com/digitalaidseattle/seattle-humane-app/assets/69748713/e76ce966-fd53-42eb-af7b-668576578b8a)

### Medium screen view
![Screenshot 2024-03-13 at 1 33 24 AM](https://github.com/digitalaidseattle/seattle-humane-app/assets/69748713/35e38bc2-0b4e-4313-92d5-230aeffda17a)

### Large screen view
![Screenshot 2024-03-13 at 1 34 55 AM](https://github.com/digitalaidseattle/seattle-humane-app/assets/69748713/3569f1cd-d14d-4611-b31c-d68d980cd6a9)

### Hover effect
![Screenshot 2024-03-13 at 1 34 31 AM](https://github.com/digitalaidseattle/seattle-humane-app/assets/69748713/aefb46bc-8c70-4d42-8b1b-ba19e311133e)

### Active route styling 
![Screenshot 2024-03-13 at 1 33 51 AM](https://github.com/digitalaidseattle/seattle-humane-app/assets/69748713/2fd47393-bae4-4a5a-ae07-b7bdbd988e0b)


## UI accessibility checklist
If your PR includes UI changes, please utilize this checklist:

- [ ] Checked with axe DevTools and addressed Critical and Serious issues.

## QA Instructions
- Seattle Humane icon and dashboard should go to the home page "/"
- Animals Button/icon should go to route "/animals"
- Clients Button/icon should go to route "clients
- The new Request button/icon should open up the request modal
- Profile button/icon should sign you out
- When hovering on these buttons red color effects should occur
- When you are on the operating routes greenish highlight border and crimson color should occur on the button that is affiliated with that route.

## Added/updated tests?
We encourage you to keep the code coverage percentage at 80% and above.

- [x]  have not been included_
- [x]  I need help with writing tests. If unit tests are necessary. so far my changes did not appear or effect current coverage

![Screenshot 2024-03-13 at 1 57 20 AM](https://github.com/digitalaidseattle/seattle-humane-app/assets/69748713/6693f9cb-ddcd-4dd6-b9c8-b6a0d4a21ab8)


[optional] Are there any post-deployment tasks we need to perform?
It is stated above about the test...
